### DISCUSSION TITLE
2025年の抱負

### DISCUSSION THREAD

**Discussion by hrmcngs** [hrmcngs/minecraft_armor_weapon#11] [Announcements]
受験などに支障が出ない程度にすごめの更新したいです

**Comment by aya526dev** [comment]
私は、みんなが使いやすいものを作りたいと思っています
受験についてはノーコメントでお願いします

**Comment by Renkon65** [comment]
回数は少ないと思うけど、頑張って更新したいです(内容は知らんけど
あと、いろんなとこの部下へのパワハラ回数を減らしていきたいです
(本当はやってないよ！ちょーーっとやりかけただけだよ！！！)
